### ISSUE TITLE
a2x error when trying to make libcoap

### ISSUE THREAD

**Issue by bsmelo** [obgm/libcoap#26]
First of all, I'm sorry if this is on the wrong place.

After an apparently successful `./configure`, I try to `make` and get the following ERROR:

```
(...)
Patching output file 1/1
lookup cache used 891/65536 hits=7471 misses=921
finished...
make[2]: Leaving directory `/home/bruno/riot_misc/libcoap/doc'
Making all in tests
make[2]: Entering directory `/home/bruno/riot_misc/libcoap/tests'
make[2]: Nothing to be done for `all'.
make[2]: Leaving directory `/home/bruno/riot_misc/libcoap/tests'
Making all in examples
make[2]: Entering directory `/home/bruno/riot_misc/libcoap/examples'
/usr/bin/a2x --doctype manpage --format manpage coap-client.txt
a2x: ERROR: "xmllint" --nonet --noout --valid "/home/bruno/riot_misc/libcoap/examples/coap-client.xml" returned non-zero exit status 127
make[2]: *** [coap-client.5] Error 1
make[2]: Leaving directory `/home/bruno/riot_misc/libcoap/examples'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/bruno/riot_misc/libcoap'
make: *** [all] Error 2
```

Is there something I'm missing here?

**Comment by obgm** [comment]
Can you check if a2x and xmllint are present on your system?

**Comment by bsmelo** [comment]
I believe they are:

```
bruno@lain ~ $ xmllint --version
xmllint: using libxml version 20901
   compiled with: Threads Tree Output Push Reader Patterns Writer SAXv1 FTP HTTP DTDValid HTML Legacy C14N Catalog XPath XPointer XInclude Iconv ISO8859X Unicode Regexps Automata Expr Schemas Schematron Modules Debug Zlib Lzma 
bruno@lain ~ $ a2x --version
a2x 8.6.9
```

**Comment by tijuca** [comment]
Obviously you didn't show the output of the configure call.
Please try this out:
```
$ git diff
diff --git a/examples/Makefile.am b/examples/Makefile.am
index 10aaae0..0dbc96c 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -36,7 +36,7 @@ MAN5 = $(TXT5:%.txt=%.5)
 man5_MANS = $(MAN5)
 
 .txt.5:
-       $(A2X) --doctype manpage --format manpage $<
+       $(A2X) --verbose --doctype manpage --format manpage $<
 
 CLEANFILES = *.5
 endif # BUILD_DOCUMENTATION
```
Then recall the autogen.sh script, also the configure call and try again to build. This should show more output and probably what's missing. I assume it will be a missing or broken docbook.dtb.
I suggest to write to the mailing list and append the compressed logs, GitHub isn't the best place for error detection like this.

**Comment by bsmelo** [comment]
Hi @tijuca, thanks for the tips.

`git diff` didn't show anything, but when redoing `autogen.sh` > `configure` > `make` I got a new error regarding xsltproc, which I really didn't have. Everything works now, after I installed it.

Thanks, guys! And sorry for bothering; I believe we can close this.
Hi @tijuca, thanks for the tips.

`git diff` didn't show anything, but when redoing `autogen.sh` > `configure` > `make` I got a new error regarding xsltproc, which I really didn't have. Everything works now, after I installed it.

Thanks, guys! And sorry for bothering; I believe we can close this.
